src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -79,7 +79,7 @@ const WorkspaceTabs = ({
     ...(isGoogleWorkspace && !isAnalysisTabVisible() ? [{ name: 'notebooks', link: 'workspace-notebooks' }] : []),
     // the spread operator results in no array entry if the config value is false
     // we want this feature gated until it is ready for release
-    ...(isGoogleWorkspace && isAnalysisTabVisible() ? [{ name: 'analyses', link: analysisTabName }] : []),
+    ...(isAnalysisTabVisible() ? [{ name: 'analyses', link: analysisTabName }] : []),
```

Comment:
In an Azure Workspace, you might want to show the Analyses Tab unconditionally since the audience is already limited.

Reply:
Eventually, analysis tab visibility will go away entirely (likely before azure is released), so I'd rather not complicate it here.